docs/getting-started-guides/docker-multinode/master.sh

```diff
@@ -41,10 +42,10 @@ fi
 
 # Make sure master ip is properly set
 if [ -z ${MASTER_IP} ]; then
-    echo "Please export MASTER_IP in your env"
-    exit 1
+    MASTER_IP=$(hostname -I | awk '{print $1}')
```

Comment:
You're right. I confused this with another script which is actually run on a client side and requires MASTER_IP set.